apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.ts

```diff
@@ -225,21 +242,8 @@ router.post(
       throw new HttpStatusError(403, 'Access denied');
     }
 
-    const courseInstancesAllowed = await courseInstancesAllowedToLink({
-      course: await selectCourseById(course_instance.course_id),
-      authzData,
-    });
-    const hasCourseInstanceAllowed = courseInstancesAllowed.some(
-      (ci) => ci.id === course_instance.id,
-    );
-
-    const coursesAllowed = await coursesAllowedToLink({
-      user_id: authzData.authn_user.user_id,
-      is_administrator: res.locals.is_administrator,
-    });
-    const hasCourseAllowed = coursesAllowed.some((c) => c.id === course_instance.course_id);
```

Comment:
I believe these checks are rendered unnecessary by the `hasRole(...)` check above, but I'd like a second set of eyes on this.